docker/edge/http/script.sh

```diff
@@ -88,7 +91,58 @@ echo -----------------------------
 echo App ID and Passwords are $ID,$SECRET 
 echo -----------------------------
 
-for connect2 in 1 2 3
+#Ftech PEP PROXY ID 
+for connect3 in 1 2 3
+do 
+   curl --include \
+     --header "X-Auth-token: $token" \
+  "http://$IP_ADDRESS:3000/v1/applications/$ID/pep_proxies" > PEP_Detail.js
+   if [ $? -eq 0 ]; then
+      break
+   else
+      echo reconnecting...
+      continue
+    fi
+done
+
+Pep_ID=`cat PEP_Detail.js | tail -1 | jq -r '.pep_proxy.id' | tr -d '\n'`
+
+#To generate PEP PROXY Password
+
+for connect4 in 1 2 3 4
+do 
+   curl --include \
+     --request PATCH \
+     --header "Content-Type: application/json" \
+     --header "X-Auth-token: $token" \
+  "http://$IP_ADDRESS:3000/v1/applications/$ID/pep_proxies" > PEP_Details.txt 
+  if [ $? -eq 0 ]; then
+     break
+  else
+     echo reconnecting...
+     continue
+  fi
+done
+
+
+Pep_password=`cat PEP_Details.txt | tail -1 | jq . | grep "new_password" | cut -f 2 -d ":" | sed -e 's/"//g' -e 's/,//g' -e 's/\r$//g' -e 's/ //g' | tr -d '/n'`
+
+
+echo -----------------------------
+echo PEP PROXY ID is $Pep_ID and PEP PROXY PASSWORD is $Pep_password
+echo --------------------------------
+
+
+#setting up of pep-config.js
+sed -i "s/PEP_PROXY_IDM_HOST.*/PEP_PROXY_IDM_HOST \|\| '${IP_ADDRESS}',/" pep-config.js
+sed -i "s/PEP_PROXY_APP_HOST.*/PEP_PROXY_APP_HOST \|\| '${EDGE_IP_ADDRESS}',/" pep-config.js
+sed -i "s/PEP_PROXY_APP_PORT.*/PEP_PROXY_APP_PORT \|\| '8060',/" pep-config.js
+sed -i "s/PEP_PROXY_APP_ID.*/PEP_PROXY_APP_ID \|\| '${ID}',/" pep-config.js
+sed -i "s/PEP_PROXY_USERNAME.*/PEP_PROXY_USERNAME \|\| '${Pep_ID}',/" pep-config.js 
+sed -i "s/PEP_PASSWORD.*/PEP_PASSWORD \|\| '${Pep_password}',/" pep-config.js
+
+
```

Comment:
[minor] Explain why this for loop is used.

Reply:
this is used for configuring pep-proxy